src/Moq/ExpressionStringBuilder.cs

```diff
@@ -325,7 +325,7 @@ private void ToStringMethodCall(MethodCallExpression node)
 				else if (node.Method.IsPropertyIndexerSetter())
 				{
 					this.builder.Append('[');
-					AsCommaSeparatedValues(node.Arguments.Skip(paramFrom), ToString);
+					AsCommaSeparatedValues(node.Arguments.Skip(paramFrom).Take(node.Arguments.Count - paramFrom - 1), ToString);
```

Comment:
It might be nice to have some kind of `.Slice(...)` extension method, but let's first see whether there would be other use cases in the Moq code base for it. Or, perhaps wait for C# 8 ranges.